site/pages/provisional-en.hbs

```diff
@@ -249,3 +255,16 @@ if ( s.getTime() &lt; msT && msT &lt; e.getTime() ) {
 		</div>
 	</div>
 </div>
+
+<h3>Icon warning color</h3>
+<p>Adding <code>provisional icon-warning-light</code> in the class will change the color</p>
+<div class="bg-warning">
+	<p class="h1">
+		<span class="glyphicon glyphicon-warning-sign provisional icon-warning-light mrgn-rght-md"></span> Without "provisional icon-warning-light" class
```

Comment:
Should be the other way around. You put the `provisional icon-warning-light` on the **Without** example, it should be on the **With** example right below instead.